train/MED.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+
+namespace MinimumEditDistance
+{
+    public class program
+    {
+        public const int Errnum = -1;
+        public static void Main()
+        {
+            Console.WriteLine("debugging");
+        }
+        public static int MED(string str1, string str2)
+        {
+            if (str1 == null && str2 == null)
```

Comment:
I think neither of two strings can be null, so that I prefer 
str1 == null || str2 == null